frontend-react/src/shared/Tooltip.tsx

```diff
@@ -0,0 +1,215 @@
+import {
+    Placement,
+    useFloating,
+    autoUpdate,
+    flip,
+    shift,
+    useHover,
+    useFocus,
+    useDismiss,
+    useRole,
+    useInteractions,
+    useMergeRefs,
+    FloatingPortal,
+    offset,
+    OffsetOptions,
+} from "@floating-ui/react";
+import classNames from "classnames";
+import React from "react";
+
+export const TOOLTIP_POSITIONS = ["top", "bottom", "right", "left"];
```

Comment:
Could these be from the [`Side`](https://github.com/floating-ui/floating-ui/blob/master/packages/core/src/types.ts#L2) enum instead?

Reply:
Absolutely, thanks for pointing out an existing type.